src/app/components/rxjs-examples/rxjs-examples.ts

```diff
@@ -70,4 +69,15 @@ export class RxJsExamples {
       return 'active';
     }
   }
+
+  /**
+   * Next two methods are not working yet
+   */
+  canActivate() {
+    console.info('canActivate works with the new router')
```

Comment:
these lifecycle hooks were changed to use annotations [angular2/src/router/lifecycle_annotations_impl.ts](https://github.com/angular/angular/blob/master/modules/angular2/src/router/lifecycle_annotations_impl.ts)